bifrost.go

```diff
@@ -85,6 +88,28 @@ func NewRainbowBridge(bc *BridgeConfig) (RainbowBridge, error) {
 	}
 
 }
+func newPinataIPFS(g *BridgeConfig) (rainbowBridge, error) {
+	
+	if g.PinataJWT == "" {
+			return nil, &errors.BifrostError{
+				Err:       fmt.Errorf("jwt is required for authentication"),
+				ErrorCode: errors.ErrUnauthorized,
+			}
+	}
+
+		// return a new Pinata Storage Provider
+	return &pinata.PinataIPFSStorage{
+		PinataJWT: g.PinataJWT,
+		Provider:        providers[strings.ToLower(g.Provider)],
+		DefaultBucket:   g.DefaultBucket,
+		CredentialsFile: g.CredentialsFile,
+		Project:         g.Project,
+		DefaultTimeout:  g.DefaultTimeout,
+		EnableDebug:     g.EnableDebug,
+		PublicRead:      g.PublicRead,
```

Comment:
Just noting this here...
Pinata has public and private files. If `PublicRead` is set to false, then files are meant to be submarined (made private) and this, in fact, requires a different authentication API key.

Reply:
okay, i can work on this though.. or should i leave this for a new PR? and fix the issue later?